bin/vcloud-launch

```diff
@@ -2,4 +2,32 @@
 
 require 'vcloud/launcher'
 
-Vcloud::Launcher::Cli.new(ARGV).run
+banner_text = <<EOF
+vcloud-launch takes a configuration describing a vCloud Org,
```

Comment:
Okay, I'm fine with it being a separate PR, but ultimately it should be re-phrased around the VMware approved spelling.